src/sqs.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash-firecloud';
+import aws from 'aws-sdk';
+
+export let getNameFromArn = _.memoize(function(queueArn) {
+  let name = /arn:aws:sqs:.*:.*:(.*)/.exec(queueArn)[1];
+  return name;
+});
+
+export let getUrl = _.memoize(async function(queueName) {
```

Comment:
I would suggest that we pass `sqs` instance as a named arg together with `queueName`. It can then have a default value of `new aws.SQS()`, so like
```
export let getUrl = _.memoize(async function({queueName, sqs = new aws.SQS()}) {
...
```

Reply:
I actually had the thought but i couldn't come up with a reason for it. Credentials maybe, now when i think about it.
Will do it, and i guess it is good to do it for all other funs in aws-util-firecloud

Reply:
I so your comment about discoverability after i pushed this. I guess that can be addressed properly via typescript defs.

Anything i can do to allure you into writing defs for this repi like you did for lodash-firecloud? 🤔🤗

Reply:
done, except you cannot memoize now (you can, but it makes too complex. the caller can take care of it

Reply:
> Anything i can do to allure you into writing defs for this repi like you did for lodash-firecloud? 🤔🤗

I can re-write it to typescript ;)

Reply:
I have to say that without memoization, it has not so much benefit anymore...

Reply:
1. memoization is an optimization, simply cannot be core functionality
2. one liner vs 10+, useful in generic places sounds like enough